specification/attestation/resource-manager/Microsoft.Attestation/attestation.json

```diff
@@ -0,0 +1,429 @@
+            "properties": {
+                "name": {
+                    "type": "string",
+                    "description": "Name of the operation."
+                },
+                "display": {
+                    "$ref": "#/definitions/OperationsDisplayDefinition",
+                    "description": "Display object with properties of the operation."
+                }
+            }
+        },
+        "OperationsDisplayDefinition": {
+            "type": "object",
+            "description": "Display object with properties of the operation.",
+            "properties": {
+                "provider": {
+                    "type": "string",
+                    "description": "Resource provider of the operation."
+                },
+                "resource": {
+                    "type": "string",
+                    "description": "Resource for the operation."
+                },
+                "operation": {
+                    "type": "string",
+                    "description": "Short description of the operation."
+                },
+                "description": {
+                    "type": "string",
+                    "description": "Description of the operation."
+                }
+            }
+        },
+        "AttestationProvider": {
+            "type": "object",
+            "description": "Attestation service response message.",
+            "required": [
+                "properties"
+            ],
+            "x-ms-azure-resource": true,
+            "properties": {
+                "id": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "The id of the ARM resource, e.g. \"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Attestation/attestationProviders/{providerName}\""
+                },
+                "name": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "The name of the  attestation service."
+                },
+                "type": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "Microsoft.Attestation/attestationProviders"
+                },
+                "properties": {
+                    "type": "object",
+                    "$ref": "#/definitions/StatusResult",
+                    "description": "Describes Attestation service status.",
+                    "x-ms-client-flatten": true
+                }
+            }
+        },
+        "StatusResult": {
+            "type": "object",
+            "description": "Status of attestation service.",
+            "required": [
+                "status"
+            ],
+            "properties": {
+                "status": {
+                    "type": "string",
+                    "description": "Status of attestation service."
```

Comment:
The GET response should mirror the PUT request body + any read-only parameters. attestationPolicy is missing from this response body. 

Is `status` a read-only or writeable property? Is there an enum of allowed string values?  Can the property name be a bit more specific?

Reply:
add
                    "enum": [
                        "Ready",
                        "NotReady",
                        "Error"
                        ]